src/posts/getting-started-talkback.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Getting started with TalkBack on Android
+description: "A brief overview of getting started with TalkBack version 13.1 on Android."
+category: Assistive Technology
+author: Cooper Hollmaier
+date: 2023-02-12
+further_reading:
+  - title: "Android Accessibility Overview"
+    url: https://support.google.com/accessibility/android/answer/6006564?hl=en
+    source: Google
+  - title: "TalkBack: Hear Your Screen Out Loud"
+    url: https://support.google.com/accessibility/android/topic/3529932?hl=en&ref_topic=9078845
+    source: Google
+  - title: "TalkBack on Android Testing Steps"
+    url: https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/talkback-android.html
+    source: BBC
+tags:
+  - assistivetechnology
+---
+
+## Introduction to TalkBack
+
+TalkBack is assistive technology that helps users with visual disabilities access native mobile applications and the web by providing audible and haptic responses to touchscreen gestures. It is the default screen reader software included on Google touchscreen devices running the Android operating system.
+
+As of 2017, TalkBack is part of a family of software products in the Android Accessibility Suite including:
+- Accessibility Menu
+- Select to Speak
+- Switch Access
+
+The Android Accessibility Suite comes pre-installed on devices running Android 9 or greater.
+
+This article is an introduction to TalkBack version 13.1. We will walk you through the basics of enabling the software and navigating websites and applications with it.
+
+## Getting Started
+
+There are three ways to activate Talkback:
+1. From the settings menu: Accessibility > TalkBack > Toggle On
+2. On the physical device: Press and hold both volume keys for three seconds
+3. Via the Google Assistant: "Hey, Google. Turn on TalkBack"
+
+Once TalkBack is on, it will begin speaking instructions to you. While there are more robust features to explore, the basic commands you should know are:
+
+- Tapping the screen selects an item
+- Double tapping the screen activates your selection
+- When using a gesture, TalkBack requires you use 2 fingers instead of one
+
+## Navigation Modes
+
+There are at least three recognized ways to navigate with TalkBack: touch, swipe, and voice commands.
+
+### Navigating by touch
+
+To navigate by touch, touch anywhere on the screen, selecting any element. It will be highlighted by a green border box similar to what you would find when using a keyboard to navigate a web page. With pressure still applied, slowly drag your finger around the screen. As you move around the screen, TalkBack will jump to the next available item and announce its contents aloud. Once you've found the content you'd like to interact with, double-tap the screen.
```

Comment:
```suggestion
To navigate by touch, touch anywhere on the screen, selecting any element. It will be highlighted by a green border box similar to what you would find when using a keyboard to navigate a web page. 

With pressure still applied, slowly drag your finger around the screen. As you move around the screen, TalkBack will jump to the next available item and announce its contents aloud. Once you've found the content you'd like to interact with, double-tap the screen.
```

Two thoughts, so two paragraphs here. First is how to activate and what it looks like when successful, then what to then do next.